test/scripts/limit-swap-test.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+date '+limit-swap-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+
+TEMPDIR=$(mktemp -d)
+trap "rm -rf $TEMPDIR" 0
+
+NETDIR=${TEMPDIR}/net
+
+if [ ! -z $BINDIR ]; then
+    export PATH=${BINDIR}:${PATH}
+fi
+
+goal network create -r ${NETDIR} -n tbd -t ${GOPATH}/src/github.com/algorand/go-algorand/test/testdata/nettemplates/TwoNodes50EachFuture.json
+
+goal network start -r ${NETDIR}
+
+# replaces prior trap0
+trap "goal network stop -r ${NETDIR}; rm -rf ${TEMPDIR}" 0
+
+export ALGORAND_DATA=${NETDIR}/Node
+
+ACCOUNT=$(goal account list|awk '{ print $3 }')
+
+
+goal asset create --creator ${ACCOUNT} --name bogocoin --unitname bogo --total 1000000000000
+
+ASSET_ID=$(goal asset info --creator $ACCOUNT --asset bogo|grep 'Asset ID'|awk '{ print $3 }')
+
+# Asset ID:         5
+
+echo "closeout part a, Algo trader"
+# quick expiration, test closeout
+
+ROUND=$(goal node status | grep 'Last committed block:'|awk '{ print $4 }')
+TIMEOUT_ROUND=$((${ROUND} + 2))
+
+sed s/TMPL_ASSET/${ASSET_ID}/g < ${GOPATH}/src/github.com/algorand/go-algorand/tools/teal/templates/limit-order-a.teal.tmpl | sed s/TMPL_SWAPN/31337/g | sed s/TMPL_SWAPD/137/g | sed s/TMPL_TIMEOUT/${TIMEOUT_ROUND}/g | sed s/TMPL_OWN/${ACCOUNT}/g | sed s/TMPL_FEE/100000/g | sed s/TMPL_MINTRD/10000/g > ${TEMPDIR}/limit-order-a.teal
+
+ACCOUNT_ALGO_TRADER=$(goal clerk compile ${TEMPDIR}/limit-order-a.teal -o ${TEMPDIR}/limit-order-a.tealc|awk '{ print $2 }')
+
+# setup trader with Algos
+goal clerk send --amount 100000000 --from ${ACCOUNT} --to ${ACCOUNT_ALGO_TRADER}
+
+goal node wait
+
+goal clerk send -a 0 -t AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ -c ${ACCOUNT} --from-program ${TEMPDIR}/limit-order-a.teal
```

Comment:
Nit: Declare at the beginning of the test script `ZERO_ADDR=AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ` and then just reuse `${ZERO_ADDR}` everywhere.